python-sdk/tests_integration/astro_deploy/deploy.sh

```diff
@@ -66,6 +66,11 @@ function deploy(){
 
     # Build image and deploy
     BUILD_NUMBER=$(awk 'BEGIN {srand(); print srand()}')
+
+    # Enforce registry name to be in lowercase
+    docker_registry_astro=$(echo $docker_registry_astro | tr '[:upper:]' '[:lower:]')
```

Comment:
do we need to enforce this from our side?

Reply:
ok, I was thinking that docker_registry_astro, deployment_id, deployment_id should have been generated in the right format in Astro, if we are changing to a lowercase here by ourselves then there might be the value which we are setting is not correct. But if we are able to deploy to both (multi/single worker) instances with these changes I'm ok with it. Do we have some job which has deployed it successfully??

Reply:
or if have tested locally, that should also be fine?